src/fitting/fitting.ts

```diff
@@ -98,7 +98,7 @@ function fittingHeader(md): void {
 
 function fittingMathBlock(md): void {
   const { marp_math_block } = md.renderer.rules
-  if (!marp_math_block) return
+  if (!marp_math_block || marp_math_block.scaled) return
```

Comment:
When `marp_math_block` renderer has marked the output is already scaled, fitting plugin won't override existed plugin.